docs/backends/http4s.md

```diff
@@ -35,6 +35,6 @@ Please note that:
 
 Instead, all custom timeout configuration should be done by creating a `org.http4s.client.Client[F]`, using e.g. `org.http4s.client.blaze.BlazeClientBuilder[F]` and passing it to the appropriate method of the `Http4sBackend` object.
 
-The backend supports streaming using fs2. For usage details, see the documentation on [streaming using fs2](fs2.md#streaming).
+The backend supports streaming using fs2. For usage details, see the documentation on [streaming using fs2](https://sttp.softwaremill.com/en/latest/backends/fs2.html#streaming).
```

Comment:
maybe let's just keep a link to `fs2.md` to avoid hard-coding links to our own docs